worked-benchmarks.yml

```diff
@@ -14,10 +14,10 @@ projects:
     build: mvn install
     clean: mvn clean
 
-  jReactPhysics3D:
-    giturl: https://github.com/opprop-benchmarks/jReactPhysics3D.git
-    build: mvn install
-    clean: mvn clean
+#  jReactPhysics3D:
```

Comment:
Could you add a comment and/or issue reference here to make clear why this is commented out?
Or remove the project here and add that comment in the other .yml file?

Reply:
Get it.